tests/testthat/test-correlation.R

```diff
@@ -113,9 +113,9 @@ test_that("Correlation Table results match", {
 
 test_that("Correlation Table hypothesis correlated positively match", {
   options <- jaspTools::analysisOptions("Correlation")
-  options$confidenceIntervals <- TRUE
-  options$displayPairwise <- TRUE
-  options$hypothesis <- "correlatedPositively"
+  options$ci <- TRUE
+  options$pairwiseDisplay <- TRUE
+  options$alternative <- "correlatedPositively"
```

Comment:
Don't we want something standardised here as well? R's default suggests "greater", "twoSided", "less"